packages/vite/src/client.ts

```
@@ -8,9 +8,12 @@
 import { getPort } from 'get-port-please'
 import { joinURL, withoutLeadingSlash } from 'ufo'
 import { defu } from 'defu'
-import { env, nodeless } from 'unenv'
+import { defineEnv } from 'unenv'
+import { resolveModulePath } from 'exsolve'
 import { defineEventHandler, handleCors, setHeader } from 'h3'
 import type { ViteConfig } from '@nuxt/schema'
+import { resolveModulePath } from 'exsolve'
```

Biome (1.9.4)

[error] Shouldn't redeclare 'resolveModulePath'. Consider to delete it or rename it.

'resolveModulePath' is defined here:


(lint/suspicious/noRedeclare)